service/account/providers/aws/account.go

```diff
@@ -0,0 +1,182 @@
+package aws
+
+import (
+	"context"
+	"encoding/json"
+	"github.com/spotinst/spotinst-sdk-go/spotinst"
+	"github.com/spotinst/spotinst-sdk-go/spotinst/client"
+	"github.com/spotinst/spotinst-sdk-go/spotinst/util/jsonutil"
+	"github.com/spotinst/spotinst-sdk-go/spotinst/util/uritemplates"
+	"io/ioutil"
+	"net/http"
+	"time"
+)
+
+type Account struct {
+	ID                 *string `json:"id,omitempty"`
+	Name               *string `json:"name,omitempty"`
+	OrganizationId     *string `json:"organizationId,omitempty"`
+	AccountId          *string `json:"accountId,omitempty"`
+	CloudProvider      *string `json:"cloudProvider,omitempty"`
+	ProviderExternalId *string `json:"providerExternalId,omitempty"`
+
+	// Read-only fields.
+	CreatedAt *time.Time `json:"createdAt,omitempty"`
+	//UpdatedAt *time.Time `json:"updatedAt,omitempty"`
```

Comment:
are we getting these in response, if not remove

Reply:
removed commented code